The catalog app crashes when the Help component is clicked.

**Describe the bug**
```
FATAL EXCEPTION: main (Ask Gemini)
                                                                                                    Process: com.google.android.fhir.catalog, PID: 20524
                                                                                                    java.lang.RuntimeException: An exception happened in constructor of class com.google.android.fhir.datacapture.QuestionnaireViewModel
                                                                                                    	at androidx.lifecycle.SavedStateViewModelFactoryKt.newInstance(SavedStateViewModelFactory.kt:232)
                                                                                                    	at androidx.lifecycle.SavedStateViewModelFactory.create(SavedStateViewModelFactory.kt:131)
                                                                                                    	at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.kt:187)
                                                                                                    	at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.kt:153)
                                                                                                    	at androidx.lifecycle.ViewModelLazy.getValue(ViewModelLazy.kt:53)
                                                                                                    	at androidx.lifecycle.ViewModelLazy.getValue(ViewModelLazy.kt:35)
                                                                                                    	at com.google.android.fhir.datacapture.QuestionnaireFragment.getViewModel(QuestionnaireFragment.kt:52)
                                                                                                    	at com.google.android.fhir.datacapture.QuestionnaireFragment.onViewCreated(QuestionnaireFragment.kt:103)
                                                                                                    	at androidx.fragment.app.Fragment.performViewCreated(Fragment.java:3147)
                                                                                                    	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:588)
                                                                                                    	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
                                                                                                    	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
                                                                                                    	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1845)
                                                                                                    	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
                                                                                                    	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3042)
                                                                                                    	at androidx.fragment.app.FragmentManager.dispatchViewCreated(FragmentManager.java:2945)
                                                                                                    	at androidx.fragment.app.Fragment.performViewCreated(Fragment.java:3148)
                                                                                                    	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:588)
                                                                                                    	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
                                                                                                    	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
                                                                                                    	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1845)
                                                                                                    	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
                                                                                                    	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:565)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:958)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:255)
                                                                                                    	at android.os.Looper.loop(Looper.java:364)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8979)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:572)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1053)
                                                                                                    Caused by: java.lang.IllegalArgumentException: List has more than one element.
                                                                                                    	at kotlin.collections.CollectionsKt___CollectionsKt.single(_Collections.kt:610)
                                                                                                    	at com.google.android.fhir.datacapture.extensions.MoreQuestionnaireResponsesKt.packRepeatedGroups(MoreQuestionnaireResponses.kt:73)
                                                                                                    	at com.google.android.fhir.datacapture.extensions.MoreQuestionnaireResponsesKt.packRepeatedGroups(MoreQuestionnaireResponses.kt:76)
                                                                                                    	at com.google.android.fhir.datacapture.extensions.MoreQuestionnaireResponsesKt.packRepeatedGroups(MoreQuestionnaireResponses.kt:56)
                                                                                                    	at com.google.android.fhir.datacapture.QuestionnaireViewModel.<init>(QuestionnaireViewModel.kt:168)
                                                                                                    	at java.lang.reflect.Constructor.newInstance0(Native Method)
                                                                                                    	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
                                                                                                    	at androidx.lifecycle.SavedStateViewModelFactoryKt.newInstance(SavedStateViewModelFactory.kt:226)
                                                                                                    	at androidx.lifecycle.SavedStateViewModelFactory.create(SavedStateViewModelFactory.kt:131) 
                                                                                                    	at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.kt:187) 
                                                                                                    	at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.kt:153) 
                                                                                                    	at androidx.lifecycle.ViewModelLazy.getValue(ViewModelLazy.kt:53) 
                                                                                                    	at androidx.lifecycle.ViewModelLazy.getValue(ViewModelLazy.kt:35) 
                                                                                                    	at com.google.android.fhir.datacapture.QuestionnaireFragment.getViewModel(QuestionnaireFragment.kt:52) 
                                                                                                    	at com.google.android.fhir.datacapture.QuestionnaireFragment.onViewCreated(QuestionnaireFragment.kt:103) 
                                                                                                    	at androidx.fragment.app.Fragment.performViewCreated(Fragment.java:3147) 
                                                                                                    	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:588) 
                                                                                                    	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272) 
                                                                                                    	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943) 
                                                                                                    	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1845) 
                                                                                                    	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782) 
                                                                                                    	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3042) 
                                                                                                    	at androidx.fragment.app.FragmentManager.dispatchViewCreated(FragmentManager.java:2945) 
                                                                                                    	at androidx.fragment.app.Fragment.performViewCreated(Fragment.java:3148) 
                                                                                                    	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:588) 
                                                                                                    	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272) 
                                                                                                    	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943) 
                                                                                                    	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1845) 
                                                                                                    	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782) 
                                                                                                    	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:565) 
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:958) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:255) 
                                                                                                    	at android.os.Looper.loop(Looper.java:364) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8979) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:572) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1053) 

```

**Component**
Core library, SDC library, reference app, or SDC gallery app

**To Reproduce**
Steps to reproduce the behavior:
1. Go to catalog app.
2. Click on Help component 
3. App exits with force close exception
4. See error

**Expected behavior**
On clicking of Help component, catalog app should not be crashed.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Smartphone (please complete the following information):**
 - Device: Nokia 3.4
 - Android version: 12
 - Build number: [e.g. Settings -> About phone -> Build number]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

**Would you like to work on the issue?**
Please state if this issue should be assigned to you or who you think could help to solve this issue.

--- a/catalog/src/main/assets/component_auto_complete.json
+++ b/catalog/src/main/assets/component_auto_complete.json
@@ -82,7 +82,7 @@
         },
         {
           "valueCoding": {
-            "code": "hypertension",
+            "code": "hbp",
             "display": "High Blood Pressure"
           }
         },

--- a/catalog/src/main/assets/component_auto_complete_with_validation.json
+++ b/catalog/src/main/assets/component_auto_complete_with_validation.json
@@ -7,6 +7,26 @@
       "type": "choice",
       "repeats": true,
       "required": true,
+      "item": [
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-displayCategory",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-display-category",
+                    "code": "instructions"
+                  }
+                ]
+              }
+            }
+          ],
+          "linkId": "instruction",
+          "text": "Try typing Asthma, Chronic Lung Disease, Depression, Diabetes, Hypertension, High Blood Pressure, or High Cholesterol",
+          "type": "display"
+        }
+      ],
       "extension": [
         {
           "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
@@ -27,7 +47,8 @@
           "valueCoding": {
             "code": "asthma",
             "display": "Asthma"
-          }
+          },
+          "initialSelected": true
         },
         {
           "valueCoding": {
@@ -55,7 +76,7 @@
         },
         {
           "valueCoding": {
-            "code": "hypertension",
+            "code": "hbp",
             "display": "High Blood Pressure"
           }
         },
@@ -65,13 +86,7 @@
             "display": "High Cholesterol"
           }
         }
-      ],
-      "initial": {
-        "valueCoding": {
-          "code": "asthma",
-          "display": "Asthma"
-        }
-      }
+      ]
     }
   ]
 }

--- a/catalog/src/main/assets/component_help.json
+++ b/catalog/src/main/assets/component_help.json
@@ -51,7 +51,7 @@
               }
             }
           ],
-          "linkId": "1.3",
+          "linkId": "1.1",
           "text": "Select one",
           "type": "display"
         },
@@ -69,7 +69,7 @@
               }
             }
           ],
-          "linkId": "1.3",
+          "linkId": "1.2",
           "text": "Only one answer can be selected. If none apply, skip the question.",
           "type": "display"
         }

